cmd/root.go

```diff
@@ -51,45 +52,57 @@ func Execute() error {
 
 func classRun(cmd *cobra.Command, args []string) error {
 	s, err := status.Initialize()
-	var tableData [][]string
-
-	class, err := strconv.Atoi(args[0])
 	if err != nil {
-		ok := false
-		if class, ok = status.CodeClassFromName(args[0]); !ok {
-			return fmt.Errorf("class: please, give a numerical value; %s", err)
-		}
+		return fmt.Errorf("class: Unable to initialize status due to: %s", err)
 	}
```

Comment:
@afh, I think you messed up the rebase of main... you removed your work from https://github.com/sterchelen/hssp/pull/8

Reply:
🤦 The joy of git merge/rebase. Thanks for bringing this to my attention, @sterchelen. I took the opportunity to improve the code a bit, so that it also allows to specify a code with the `class` subcommand…

Reply:
Thanks @afh but I don't see the usage of giving a code number to have the whole class. If you know the code number you know its class. Second argument, it's not user-friendly neither expected at all.
Could you remove this feature please? Otherwise everything is clear on my side 👍🏼.

Reply:
Sure thing, @sterchelen :+1: